plyer/platforms/linux/keystore.py

```diff
@@ -1,6 +1,6 @@
 try:
     import keyring
-except:
+except Exception:

```

Comment:
Ok, not sure if we can do better there, but that would be nice, excepting on `Exception` is equivalent to just using a bare `except`, so it's a bit bad to do that to follow a style guide.